api/routes/wavFile.py

```diff
@@ -0,0 +1,52 @@
+from flask import Blueprint, jsonify, abort, request, send_file
+from ravel.api.models.wavFile import WavFile
+from ravel.api import db
+from io import BytesIO
+
+wav = Blueprint('wav', __name__)
+wav.config = {}
+base_wavFile_url = '/api/wav'
+
+@wav.route('%s'% base_wavFile_url, methods=['POST'])
+def create_wavFile(): 
+
+    wavFile = request.files['file']
+    binary = wavFile.read()
+    wavFile = WavFile.query.filter_by(binary=binary).first()
```

Comment:
True true, Its fast but could take longer with server load. I'll look into a way to fixing this. The problem I ran into is not knowing how to send a file and body over http. If I could do both then I would compare by name

Reply:
I would consider maybe trying a hash comparison of the hashed binary. It would probably be a lot faster to check two hashes for equality than two full binaries.

Reply:
dope idea